rllib/evaluation/rollout_worker.py

```diff
@@ -163,7 +162,7 @@ def _update_env_seed_if_necessary(
 
 
 @DeveloperAPI
-class RolloutWorker(ParallelIteratorWorker, FaultAwareApply):
+class RolloutWorker(ParallelIteratorWorker, EnvRunner):
```

Comment:
oh ok. So RolloutWorker is a special EnvRunner. I see now why you made certain choices. But I think still the parent class may not need get_node_ip or those other APIs.

Reply:
Correct:
Base: EnvRunner --> RolloutWorker
                              \-> DreamerEnvRunner

True, maybe these extra APIs we don't need (leave them in RW). Lemme think about these some more.

Reply:
Moved `get_node_ip`, `get_host`, `find_free_port` back into RW-only. Not part of this base EnvRunner API anymore.